cookbooks/ey-cron/recipes/default.rb

```diff
@@ -12,14 +12,24 @@
   only_if node['dna']['crons'].empty?
 end
 
-cron_header = <<-CRON
-# begin-ey-cron-header This is a delimeter. DO NOT DELETE
+cron_header = ""
+if node.engineyard.environment.ruby?
```

Comment:
1. Remove empty space on the left of file output
2. Add new line as the last line on output 

It's now producing:

```
  # begin-ey-cron-header This is a delimeter. DO NOT DELETE

  PHP_ENV="production"
  # end-ey-cron-header This is a delimeter. DO NOT DELETE
# Chef Name: ey-snapshots
```

Should be 

```
# begin-ey-cron-header This is a delimeter. DO NOT DELETE

PHP_ENV="production"
# end-ey-cron-header This is a delimeter. DO NOT DELETE

# Chef Name: ey-snapshots
```

Reply:
Fixed 1 but not 2. If I add a newline after the delimiter, each chef run will add another newline.

Reply:
That works too!